### PR TITLE
testsuite: Add test for skipping sosreport dirs

### PR DESCRIPTION
**Pull request by mkutlak**
Related to:
https://github.com/abrt/libreport/pull/571

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

File: tests/runtests/skip-sosreport-dir/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+        rlRun "rm $crash_PATH/sosreport.log" 0 "Remove FAKE sosreport.log file"
+
+        rlRun "abrt-action-trim-files -vvv -d 1M:$ABRT_CONF_DUMP_LOCATION &> $TmpDir/trim-files-delete-output" 0
+        rlRun "cat $TmpDir/trim-files-delete-output"
+
+        rlAssertNotExists "$crash_PATH"
+        rlAssertGrep "deleting '$(basename $crash_PATH)'" "$TmpDir/trim-files-delete-output"
+
+        rlRun "abrt-cli rm $crash_PATH" 1 "Remove crash directory"
```
Comment (ernestask) [inline tests/runtests/skip-sosreport-dir/runtest.sh]: Can you use `remove_problem_directory` here?

Comment (ernestask) [reply tests/runtests/skip-sosreport-dir/runtest.sh]: I painstakingly replaced this everywhere should we ever change the CLI again.

Comment (ernestask) [reply tests/runtests/skip-sosreport-dir/runtest.sh]: Of course, in cases where you’re removing multiple directories, this won’t work, but oh well.

Comment (mkutlak) [reply tests/runtests/skip-sosreport-dir/runtest.sh]: > Can you use `remove_problem_directory` here?

I am testing for a failure (1). 

I could maybe modify the function to accept different $crash_PATHs and return codes.

Comment (mkutlak) [reply tests/runtests/skip-sosreport-dir/runtest.sh]: > > Can you use `remove_problem_directory` here?
> 
> I am testing for a failure (1).

I should change the comment to make it more clear.

Comment (ernestask) [reply tests/runtests/skip-sosreport-dir/runtest.sh]: Oops, pardon my reading comprehension.

Comment (ernestask) [reply tests/runtests/skip-sosreport-dir/runtest.sh]: I think it’s fine how it is, no need for anything else.

File: tests/runtests/skip-sosreport-dir/runtest.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+SOSREPORT_EVENT="/etc/libreport/events.d/sosreport_event.conf"
+
+rlJournalStart
+    rlPhaseStartSetup
+        check_prior_crashes
+        TmpDir=$(mktemp -d)
+        pushd "$TmpDir" || exit
+
+        # generate large file to fill problem directories space
+        dd if=/dev/urandom of=hugh_mongous count=13337 bs=1024
```
Comment (ernestask) [inline tests/runtests/skip-sosreport-dir/runtest.sh]: >hugh_mongous

I see what you did there.

**Pull request merged**